packages/auth/src/graphql/resolvers/grant.test.ts

```diff
@@ -220,6 +221,96 @@ describe('Grant Resolvers', (): void => {
         expect(query.edges).toHaveLength(0)
         expect(grants).toHaveLength(2)
       })
+
+      test('finalizationReason: in', async (): Promise<void> => {
+        const grants: GrantModel[] = []
+        for (let i = 0; i < 2; i++) {
+          const grant = await createGrant(deps)
+          await grant.$query().patch({
+            state: GrantState.Finalized,
+            finalizationReason: GrantFinalization.Revoked
+          })
+          grants.push(grant)
+        }
+
+        const filter = {
+          finalizationReason: {
+            in: [GrantFinalization.Revoked]
+          }
+        }
+
+        const query = await appContainer.apolloClient
+          .query({
+            query: gql`
+              query grants($filter: GrantFilter) {
+                grants(filter: $filter) {
+                  edges {
+                    node {
+                      id
+                      state
+                    }
+                    cursor
+                  }
+                }
+              }
+            `,
+            variables: { filter }
+          })
+          .then((query): GrantsConnection => {
+            if (query.data) {
+              return query.data.grants
+            } else {
+              throw new Error('Data was empty')
+            }
+          })
+        expect(query.edges).toHaveLength(2)
+        expect(grants).toHaveLength(2)
+      })
+
+      test('finalizationReason: not in', async (): Promise<void> => {
```

Comment:
we're expecting either or, correct? not both `in` and `not`?

Reply:
that's correct. `finalizationReason: notIn` would be probably be better.

Reply:
changed the name here